### DISCUSSION TITLE
Suggestions for the Clozure CL 1.12 release notes

### DISCUSSION THREAD

**Discussion by marktmiller** [Clozure/ccl#427] [General]
Overall, I found these instructions worked as expected, but I had a suggestion, as I was new to doing the build process.

After running the make steps for building the kernel, I was a bit lost about what to do next. The instructions just say "start the lisp and evaluate (rebuild-ccl :clean t)". I was wondering how I should start lisp up. After doing some poking around, I figured out I should do:
`cd ../..`
`./dx86cl64`
That took me back to ccl-dev, where I found dx86cl64. Evaluating the lisp expression in that seemed to work (no errors). I just would've appreciated seeing those steps included in the instructions.

**Comment by informatimago** [comment]
Or you can also install the ccl or ccl64 script in your PATH and just type `ccl`.

**Reply by marktmiller** [reply]
My situation was that I had installed Clozure several years ago from an install package (maybe in a tar file). Everything was pre-compiled, and in the ccl directory. I have not been a frequent user of it. I used to just cd into the ccl directory, and run the script from there. I didn't have it in my path. Either that, or I ran Clozure from the IDE, in Applications.

What I had been used to with upgrading to a newer version of anything on my Mac was you either overwrite the original installation, or delete it, and start fresh with the new one.

With this installation, I came to find out the make file compiled the kernel to ~/ccl-dev (I was looking for it in a bin directory under ccl-dev).

Anyway, cleared up now. I also compiled the IDE, and it works.
